setup.py

```diff
@@ -0,0 +1,35 @@
+from setuptools import setup, find_namespace_packages
+
+install_requires = ["astropy", "astroquery"]
+tests_require = ["pytest", "pytest-cov", "pytest-flake8"]
+dev_requires = install_requires + tests_require + ["documenteer[pipelines]"]
+scm_version_template = """# Generated by setuptools_scm
+__all__ = ["__version__"]
+
+__version__ = "{version}"
+"""
+
+setup(
+    name="ts_scriptqueue",
+    description="A Commandable SAL Component (CSC) that runs Python scripts..",
+    use_scm_version={
+        "write_to": "python/lsst/ts/scriptqueue/version.py",
+        "write_to_template": scm_version_template,
+    },
+    setup_requires=["setuptools_scm", "pytest-runner"],
+    install_requires=install_requires,
+    package_dir={"": "python"},
+    packages=find_namespace_packages(where="python"),
+    tests_require=tests_require,
```

Comment:
I think we are discontinuing the use of `tests_require`. At least that's what @couger01 told me, and I've been removing it from my packages.